ParametersXmlAddin/ParametersXmlAddin/source.extension.vsixmanifest

```diff
@@ -13,12 +13,8 @@ If the parameters.xml already exists in the folder (even if it is not a file in
         <Icon>Resources/Package.ico</Icon>
     </Metadata>
     <Installation InstalledByMsi="false">
-        <InstallationTarget Version="[12.0,17.0)" Id="Microsoft.VisualStudio.IntegratedShell" />
+        <InstallationTarget Version="[14.0,17.0)" Id="Microsoft.VisualStudio.IntegratedShell" />
```

Comment:
📝 This file includes `<Prerequisites>` that prevent installation in certain early versions of Visual Studio 2017. This seems inconsistent with the general support for Visual Studio 2015, but I didn't fix it here.